gclient/src/api/calls.rs

```diff
@@ -295,7 +294,8 @@ impl GearApi {
             } = &gas_node.1
             {
                 accounts_with_reserved_funds.insert(id);
-                src_program_reserved_gas_total += value + lock.0[LockId::Reservation as usize];
+                // `2` here `LockId::Reservation` in `u8`
```

Comment:
That's unsafe code for future

Reply:
it is really hard to publish `gear-common` since it has substrate dependencies, for example `frame-support` is already at `frame-support = "21.0.0"` on crates.io

`gclient` imports `gear-common` just for this `LockId`, so I'm thinking of if we can move this `LockId` to `gear-core`?

Reply:
#2965

Reply:
so current change should be reverted, isn't it?

Reply:
no, the term of publishing `gear-common` is just for the `LockId`, we don't have this issue many more after https://github.com/gear-tech/gear/pull/2966